### PR TITLE
Add lambda to frontend

### PR DESCRIPTION
**Pull request by steveliupai**
1. Update chain info
2. Update ibc info
3. Add token icon
4. Create LAMB-ATOM pool and submit info  [tx](https://www.mintscan.io/osmosis/txs/991A5D9D3A8EAF4BCE1A3224014AB6DFA300A26993C397C0494BA46E48F3C336)

@JeremyParish69 Hi Jeremy. This is the update for lambda to frontend, please view and merge. Thank you very much!

### CODE REVIEW

File: packages/web/config/price.ts [hunk 1]
```diff
@@ -553,4 +553,17 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     spotPriceDestDenom: "uosmo",
     destCoinId: "osmosis",
   },
+  {
+    alternativeCoinId: "pool:lambda",
+    poolId: "823",
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-378" }],
```
Comment (JeremyParish69) [inline packages/web/config/price.ts:560]: In the chain registry, it says that channel 350 is the channel id, not 378. Are you able to reconsider and confirm which is the correct channel-id, please?
https://github.com/cosmos/chain-registry/blob/master/_IBC/lambda-osmosis.json

Comment (JeremyParish69) [reply packages/web/config/price.ts:560]: nvm, I saw the PR that corrects the info in the chain registry

**Review by JeremyParish69 — APPROVED** [review]

**Comment by steveliupai** [comment]
@jonator Hi Jonator, please merge, thank you so much!

File: packages/web/config/price.ts (hunk 1 above)
Comment (steveliupai) [reply packages/web/config/price.ts:560]: We have replaced expired client and set new channel

**Comment by steveliupai** [comment]
@JeremyParish69 Hi Jeremy, what should i do next to get this PR merged, we have created pool but can't not show the correct denom in frontier.

**Comment by steveliupai** [comment]
@JeremyParish69 @jonator  https://frontier.osmosis.zone/pool/826   OSMO/LAMB pool created!

**Pull request merged**